src/plugins/data_source/server/index.ts

```diff
@@ -8,7 +8,7 @@ import { PluginConfigDescriptor, PluginInitializerContext } from 'src/core/serve
 import { DataSourcePlugin } from './plugin';
 
 export const configSchema = schema.object({
-  enabled: schema.boolean({ defaultValue: false }),
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
change it back to default false. It's a switch that will finally be aligned with global config

Reply:
done